service-loadbalancer/Dockerfile

```diff
@@ -16,10 +16,12 @@
 FROM alpine:3.2
 MAINTAINER Prashanth B <beeps@google.com>
 
-RUN apk add -U haproxy bash curl && \
+RUN apk add -U pcre bash curl socat lua5.3 && \
+  wget -O /var/cache/apk/haproxy.apk https://github.com/aledbf/alpine-build-dev-haproxy/releases/download/1.6-r0/haproxy-1.6-r0.apk && \
```

Comment:
Hmm, we need to make it reproducible, perhaps you can upstream your build scripts into a /build subdir and modify the Makefile? Then it doesn't really matter where we host the tarball.